### PR TITLE
Update images.md

### PR DESCRIPTION
**Pull request by silentoplayz**
Updoot to ComfyUI installation and Flux guide.

### CODE REVIEW

File: docs/tutorial/images.md [hunk 1]
```diff
@@ -53,49 +53,49 @@ ComfyUI provides an alternative interface for managing and interacting with imag
    docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -e COMFYUI_BASE_URL=http://host.docker.internal:7860/ -e ENABLE_IMAGE_GENERATION=True -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
    ```
 
-### Configuring Open WebUI
+## Setting Up Open WebUI with ComfyUI
```
Comment (justinh-rahb) [inline docs/tutorial/images.md:56]: let's make this `###`

File: docs/tutorial/images.md [hunk 2]
```diff
@@ -53,49 +53,49 @@ ComfyUI provides an alternative interface for managing and interacting with imag
...
-1. Navigate to the **Admin Panel** > **Settings** > **Images** menu in Open WebUI.
-2. Set the `Image Generation Engine` field to `ComfyUI`.
-3. In the API URL field, enter the address where ComfyUI's API is accessible:
-   ```
-   http://<your_comfyui_address>:8188/
-   ```
-   Set the environment variable `COMFYUI_BASE_URL` to this address to ensure proper integration.
-4. Verify the connection.
-5. Save changes.
+### Setting Up Flux.1 Models:
```
Comment (justinh-rahb) [inline docs/tutorial/images.md:58]: And `####`

File: docs/tutorial/images.md [hunk 3]
```diff
@@ -53,49 +53,49 @@ ComfyUI provides an alternative interface for managing and interacting with imag
...
-COMFYUI_SCHEDULER="simple"
-COMFYUI_SD3="false"
-COMFYUI_FLUX="true" # Enables ComfyUI Flux mode.
-COMFYUI_FLUX_WEIGHT_DTYPE="fp8_e4m3fn" # Ignored if Flux is not enabled. Sets the weight precision for Flux.
-COMFYUI_FLUX_FP8_CLIP="true # Enable 8-bit precision for the Flux text encoder.
-```
+To integrate ComfyUI into Open WebUI, follow these steps:
 
-**Important Notes:**
+### Step 1: Configure Open WebUI Settings
```
Comment (justinh-rahb) [inline docs/tutorial/images.md:75]: `####`

File: docs/tutorial/images.md [hunk 4]
```diff
@@ -53,49 +53,49 @@ ComfyUI provides an alternative interface for managing and interacting with imag
...
-* An updated version of ComfyUI is required.
-* Specific model checkpoints and files must be present in your ComfyUI installation.
+1. Navigate to the **Admin Panel** in Open WebUI.
+2. Click on **Settings** and then select the **Images** tab.
+3. In the `Image Generation Engine` field, choose `ComfyUI`.
+4. In the **API URL** field, enter the address where ComfyUI's API is accessible, following this format: `http://<your_comfyui_address>:8188/`. 
+   - Set the environment variable `COMFYUI_BASE_URL` to this address to ensure it persists within the WebUI.
 
-### Model Checkpoints and Files
+### Step 2: Verify the Connection and Enable Image Generation
```
Comment (justinh-rahb) [inline docs/tutorial/images.md:83]: etc

File: docs/tutorial/images.md [hunk 5]
```diff
@@ -53,49 +53,49 @@ ComfyUI provides an alternative interface for managing and interacting with imag
...
-The following files are required for Flux mode:
+1. Ensure ComfyUI is running and verify the connection to Open WebUI. You won't be able to proceed without a successful connection.
+2. Once the connection is verified, toggle on **Image Generation (Experimental)**. More options will be presented to you.
+3. Continuye to step 3 for the final configuration steps.
 
-* Model checkpoints in both `models/checkpoints` and `models/unet` directories. You can download the Flux model(s) from the [black-forest-labs HuggingFace page](https://huggingface.co/black-forest-labs).
-* `clip_l.safetensors` in the `models/clip` directory. Download from [here](https://huggingface.co/comfyanonymous/flux_text_encoders/tree/main).
-* `t5xxl_fp16.safetensors` in the `models/clip` directory, unless `COMFYUI_FLUX_FP8_CLIP` is enabled. Download from [here](https://huggingface.co/comfyanonymous/flux_text_encoders/tree/main).
-* `ae.safetensors` in the `models/vae` directory. Download from [here](https://huggingface.co/black-forest-labs/FLUX.1-schnell/blob/main/ae.safetensors).
+### Step 3: Configure ComfyUI Settings and Import Workflow
```
Comment (justinh-rahb) [inline docs/tutorial/images.md:89]: etc

**Review by justinh-rahb — COMMENTED** [review]
🙏 Love it, just think it needs these headings brought down a size

File: docs/tutorial/images.md (hunk 5 above)
Comment (silentoplayz) [reply docs/tutorial/images.md]: Should all be fixed

**Comment by justinh-rahb** [comment]
LGTM! 👌

**Pull request merged**